server/src/events.mjs

```diff
@@ -226,6 +226,9 @@ function sendEventListenerAck(userId, ws, metadata) {
   const ackMessage = template(metadata);
   const parsedAckMessage = JSON.parse(ackMessage);
 
+  const userData = userManagement.getWsForUser(config?.dotConfig?.interactionService?.user);
+  createInteractionLog(ackMessage, metadata.method, metadata.registration.params, userData); // creating interaction log and send it to the client
```

Comment:
this condition is placed inside *createInteractionLog* the function, so instead of having it in all the places, I thought to put it inside the function itself